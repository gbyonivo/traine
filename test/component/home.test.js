import React from 'react';
import { create } from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Home from '../../src/components/home';

describe('Home', () => {
  it('should render correctly', () => {
    const actual = create(
      <MemoryRouter initialEntries={[{ pathname: '/', key: 'testKey' }]}>
        <Home/>
      </MemoryRouter>
    );
    expect(actual).toMatchSnapshot();
  });
});
