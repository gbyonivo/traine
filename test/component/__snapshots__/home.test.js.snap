// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render correctly 1`] = `
<div
  className="home"
>
  <a
    href="/current"
    onClick={[Function]}
  >
    <div
      className="current nav"
    >
      Current Weather
    </div>
  </a>
  <a
    href="/forecast"
    onClick={[Function]}
  >
    <div
      className="forecast nav"
    >
      Weather Forecast
    </div>
  </a>
</div>
`;
